src/goto-instrument/cover.cpp

```diff
@@ -1533,14 +1686,15 @@ bool instrument_cover_goals(
       symbol_table,
       goto_functions,
       criterion,
+      message_handler,
       existing_goals,
       cmdline.isset("cover-function-only"),
-      cmdline.isset("no-trivial-tests"));
+      cmdline.isset("no-trivial-tests"),
+      cmdline.get_value("cover-include-pattern"));
```

Comment:
does it make sense to allow both `cover-function-only` and `cover-include-pattern` ? Now, the former seems to be just a special case of the latter.

Reply:
`cover-function-only` is redundant, but more convenient to use. I would not remove it immediately, we can deprecate it later.